docker-compose.yml

```diff
@@ -126,6 +126,10 @@ services:
       timeout: 5s
       retries: 15
       start_period: 60s
+    volumes:
```

Comment:
Commit message nit: Instead of repeating the title as part of the body, you could have also just written "This prevents...".